lib/skinny_controllers/lookup/ensure_existence.rb

```diff
@@ -113,5 +101,33 @@ def operation_lookup(qualified_name)
       # recurse
       operation_lookup(next_lookup)
     end
+
+    private
+
+    # Decides if the constant exists or if we can travel any higher. Also,
+    # validates the qualified_name in case it is blank.
+    #
+    # @param [String] qualified_name
+    # @param [Symbol] sender The sender method
+    #
+    # @return [Array<Constant, Boolean>]
+    def lookup_helper(qualified_name, sender)
+      # Validate the name.
+      name_flag = qualified_name.blank?
+      return klass, name_flag if name_flag # return nil as the klass value
+
+      # Return if the constant exists.
+      klass = qualified_name.safe_constantize
```

Comment:
Check c362a96. But this way the `blank?` guard is being silenced.
It seems to me the only change worth keeping is 49c83e6a.